src/website/views/components/TimestampFilter/TimestampFilter.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+import Accordion from '@mui/material/Accordion';
+import AccordionSummary from '@mui/material/AccordionSummary';
+import AccordionDetails from '@mui/material/AccordionDetails';
+import Typography from '@mui/material/Typography';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import { Button, Grid } from '@mui/material';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { DateTimePicker } from '@mui/x-date-pickers/DateTimePicker';
+import styles from './timestampfilter.module.css';
+
+// FIX ELEVATION ISSUE !!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+function TimestampFilter(props) {
+  const [startDate, setStartDate] = useState('');
+  const [endDate, setEndDate] = useState('');
+
+  const dispatch = props.dispatch;
+
+  const handleApplyChange = () => {
+    let cond1 = startDate != null && endDate != null;
+    let cond2 = _parseISOString(endDate) > _parseISOString(startDate);
+
+    if (cond1 == true && cond2 == true) {
+      dispatch({
+        type: 'TIMESTAMP',
+        payload: {
+          startDate: startDate.toISOString(),
+          endDate: endDate.toISOString(),
+        },
+      });
+    }
+  };
+
+  const handleResetChange = () => {
+    setStartDate(null);
+    setEndDate(null);
+
+    dispatch({
+      type: 'TIMESTAMP',
+      payload: {
+        startDate: null,
+        endDate: null,
+      },
+    });
+  };
+
+  const handleStartChange = (newStartDate) => {
+    setStartDate(newStartDate);
+  };
+
+  const handleEndChange = (newEndDate) => {
+    setEndDate(newEndDate);
+  };
+
+  function _parseISOString(s: string) {
```

Comment:
No need to include the underscore in the function here

Reply:
done